core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java

```diff
@@ -280,7 +281,7 @@ private DataType fromCassandraDataType(
       case BLOB:
         return DataType.BLOB;
       default:
-        throw new UnsupportedTypeException(
+        throw new ExecutionException(
```

Comment:
Any particular reason for throwing a checked exception here?
Looks like it's a programs' error or user's error so should they be an assertion error or unchecked exception?

Reply:
Thank you for the discussion! I understand the point so let's leave it as it is.